RiotSwiftUI/Modules/Room/Composer/CreateActionList/Coordinator/ComposerCreateActionListCoordinator.swift

```diff
@@ -43,7 +43,11 @@ final class ComposerCreateActionListCoordinator: NSObject, Coordinator, Presenta
         viewModel = ComposerCreateActionListViewModel(initialViewState: ComposerCreateActionListViewState(actions: actions))
         view = ComposerCreateActionList(viewModel: viewModel.context)
         let hostingVC = VectorHostingController(rootView: view)
-        hostingVC.bottomSheetPreferences = VectorHostingBottomSheetPreferences(detents: [.medium])
+        hostingVC.bottomSheetPreferences = VectorHostingBottomSheetPreferences(
+            detents: [.custom(identifier: "ComposerCreateActionList", height: 360)],
```

Comment:
Seeing the hardcoded 360 there I wonder does it work well on all devices(like small ones)? Do they always fit on screen and if not can you at least scroll to the to different items?

Reply:
will test this on a SE since is the smallest device possible and add a screenshot

Reply:
![Simulator Screen Shot - iPhone SE (3rd generation) - 2022-10-17 at 13 26 14](https://user-images.githubusercontent.com/34335419/196165515-2c831def-0854-48fc-9714-e0a8473d2574.png)

Looks pretty good, ofc if in the future we add more stuff to that bottom sheet, we either revert it back to medium, and make it scrollable, or we increase that size

Reply:
OS 16 is not available yet on the CI, so the custom size detent can't be implemented yet, will have it on separate branch for when the Xcode version of the CI will be updated